### PR TITLE
Update include.md

### PR DESCRIPTION
**Pull request by tjorim**
Mimic this change from main to experimental documentation.

### CODE REVIEW

**Comment by ludeeus** [comment]
Is this the only commit thats missing?

File: documentation/publish/include.md [hunk 1]
```diff
@@ -97,5 +97,4 @@ This make sure that the files still are sorted correctly.
 
 ## After Submitting
 
-HACS contains a significant amount of repos, with new repositories added almost daily. Check the [backlog](https://github.com/hacs/default/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-asc+draft%3Afalse) before querying your PR. Updating your PR will drop it to the bottom of the list.   
-
+HACS contains a significant amount of repos, with new repositories added almost daily. Check the [backlog](https://github.com/hacs/default/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-asc+draft%3Afalse) before querying your PR.
```
Comment (ludeeus) [inline documentation/publish/include.md:100]: This was done by a manual change, it seems.
You should use git commands to pull in https://github.com/hacs/documentation/commit/78a22dc3fc26e82abfc9f28db94391e1a6c29472

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: documentation/publish/include.md (hunk 1 above)
Comment (tjorim) [reply documentation/publish/include.md:100]: You're right, I don't have access to my proper Git currently. Would also need https://github.com/hacs/documentation/commit/c84d3219df3f3270946ae54ecd9b0dffbc9fddb4.

Comment (ludeeus) [reply documentation/publish/include.md:100]: I can do a sync later today for it 👍

Comment (ludeeus) [reply documentation/publish/include.md:100]: https://github.com/hacs/documentation/commit/3a56f968b8b55fc7e2c6875ff76903205befd192

**Pull request closed without merging**